### ISSUE TITLE
Update Newtonsoft.Json dependency version

### ISSUE THREAD

**Issue by jasturgill** [sendgrid/sendgrid-csharp#744]
Are there plans to update this package to use more recent versions of the newtonsoft.json package?  It fails if any version above 9.0.1 is installed.

**Comment by thinkingserious** [comment]
Now there is :)

Thanks for bringing this to our attention.

**Comment by fabricioveronez** [comment]
I made a pull request to fix this issue

**Comment by krillehbg** [comment]
Please fix this and merge soon

**Comment by jnyrup** [comment]
@krillehbg Have you tried using `<bindingRedirect>`?

Example:
I have this dependency chain:
* SendGrid is used by MyLibrary
* MyLibrary is used by MyApplication

In the `app.config` for MyApplication I've added:
```
<dependentAssembly>
  <assemblyIdentity name="Newtonsoft.Json" publicKeyToken="30ad4fe6b2a6aeed" culture="neutral" />
  <bindingRedirect oldVersion="0.0.0.0-12.0.0.0" newVersion="12.0.0.0" />
</dependentAssembly>
```

**Comment by katiep23** [comment]
Is this still not fixed?

**Comment by childish-sambino** [comment]
This is a similar issue reported in the Twilio helper lib. Latest comment here: https://github.com/twilio/twilio-csharp/issues/422#issuecomment-666681084

tl;dr: Use binding redirects whose purpose is to resolve such dependency _conflicts_.

**Comment by frazerdavidb** [comment]
Looks like it is still not fixed. Crazy this has gone on this long. Pulled down your code and going to fix it myself.

Error:
System.IO.FileLoadException: Could not load file or assembly 'Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
File name: 'Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed'
   at SendGrid.Helpers.Mail.SendGridMessage.Serialize(Boolean useDefaultSerialization)
   at SendGrid.BaseClient.<SendEmailAsync>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at HostedFrazerAdminConsole.SendGridAPI.<sendEmailTask>d__25.MoveNext()
